gdx/src/com/badlogic/gdx/graphics/g2d/GlyphLayout.java

```diff
@@ -238,7 +237,7 @@ else if (targetWidth <= fontData.spaceXadvance * 3) //
 								adjustLastGlyph(fontData, previous);
 							}
 						} else {
-							next = wrap(fontData, run, wrapIndex, i);
+							next = wrap(fontData, run, wrapIndex);
```

Comment:
`i` is not used in the private method `wrap`

Reply:
These unused parameters are good to clean up. Leftover from when it tried to avoid looping through all the runs at the end.